llvm-10.0.1/llvm-crash-analyzer/test/Analysis/test-verify-src-dest.mir

```diff
@@ -0,0 +1,169 @@
+
+# RUN: %llvm-crash-analyzer-ta -print-dest-src-operands %s 2> %t.log
+# RUN: FileCheck %s < %t.log
+# CHECK:   $edx = crash-start MOV32rm $rcx, 1, $noreg, 0, $noreg
+# CHECK: Destination: $edx
+# CHECK: Source1: $rcx
+# CHECK: Source1 Offset: 0
+# CHECK:   ADD64mi8 $rbp, 1, $noreg, 327056915, $noreg, 1, implicit-def $eflags
+# CHECK: Destination: $rbp
+# CHECK: Destination Offset: 327056915
+# CHECK: Source1: 1
+# CHECK:   MOV32mi $rbp, 1, $noreg, -4, $noreg, 0
+# CHECK: Destination: $rbp
+# CHECK: Destination Offset: -4
+# CHECK: Source1: 0
+# CHECK:   TEST8mi $rdx, 1, $noreg, 11, $noreg, 1, implicit-def $eflags
+# CHECK: Source1: $rdx
+# CHECK: Source1 Offset: 11
+# CHECK: Source2: $rdx
```

Comment:
The MIR opcode is TEST8mi. So, I expect source2 to be an immediate value(1).  Why you chose $rdx as  source2 ?
How do you handle these TEST instruction which set flags?

Reply:
Fixed (The immediate value wasn't printed correctly). 
While there is no destination operand for an instruction like TEST8mi, the memory operand in this instruction could be invalid and hence it needs to be identified and tainted appropriately. Currently startTaint will taint source memory operands as well if present.